package.json

```diff
@@ -1,17 +1,28 @@
 {
-  "name": "databraid-react-starter",
+  "name": "transit-spa",
   "version": "1.0.0",
   "private": true,
   "dependencies": {
+    "enzyme": "^2.9.1",
```

Comment:
per Josh:

when installing packages, please only install fixed versions. If you have `npm` version 3 installed this will be the default. For example: `my-package: 1.0.1` and not `my-package:^1.0.0`. Thanks!